x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import { AppMountContext, AppMountParameters } from 'kibana/public';
+import { I18nProvider, FormattedMessage } from '@kbn/i18n/react';
+
+/**
+ * This module will be loaded asynchronously to reduce the bundle size of your plugin's main bundle.
+ */
+export function renderApp(appMountContext: AppMountContext, { element }: AppMountParameters) {
+  appMountContext.core.http.get('/api/endpoint/hello-world');
+
+  ReactDOM.render(<AppRoot />, element);
+
+  return function() {
```

Comment:
```suggestion
  return () => {
```

nit: https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md#prefer-modern-javascripttypescript-syntax


"Prefer arrow fns to fn expressions"

Reply:
curious what the reasoning is there. Seems to hinder readability (harder to scan the text and see 'function'.) It also leaves the reader wondering if there is a reason for the arrow function, in other words, has `this` been used.